experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -10,11 +10,104 @@ export strict_transform, total_transform, weak_transform, controlled_transform
 
 export prepare_smooth_center, as_smooth_local_complete_intersection, as_smooth_lci_of_cod, _non_degeneration_cover
 
+abstract type AbsProjectiveGlueing{
+                                   GlueingType<:AbsGlueing,
+                                  }
+end
+
+### getters for the essential functionality
+base_glueing(PG::AbsProjectiveGlueing) = base_glueing(underlying_glueing(PG))
+inclusion_maps(PG::AbsProjectiveGlueing) = inclusion_maps(underlying_glueing(PG))
+glueing_domains(PG::AbsProjectiveGlueing) = glueing_domains(underlying_glueing(PG))
+patches(PG::AbsProjectiveGlueing) = patches(underlying_glueing(PG))
+glueing_morphisms(PG::AbsProjectiveGlueing) = glueing_morphisms(underlying_glueing(PG))
+
+
+@Markdown.doc """
+  LazyProjectiveGlueing(
+      X::AbsProjectiveScheme,
+      Y::AbsProjectiveScheme,
+      BG::AbsGlueing,
+      compute_function::Function, 
+      glueing_data
+    )
+
+Produce a container `pg` to host a non-computed `ProjectiveGlueing` of ``X`` with ``Y``.
+
+The arguments consist of 
+
+ * the patches ``X`` and ``Y`` to be glued;
+ * a glueing `BG` of the `base_scheme`s of ``X`` and ``Y`` over which the 
+   `ProjectiveGlueing` to be computed sits;
+ * a function `compute_function` which takes a single single argument `glueing_data` 
+   of arbitrary type and actually carries out the computation; 
```

Comment:
Does the compute function compute a single glueing of one chart 
or of all charts at once? Or is the output a ProjectiveGlueing? please document accordingly.

Reply:
One glueing. Why should it be taken from within the double for-loop otherwise? I think, it's already spelled out.

Reply:
What are the charts? X and Y? but these are projective schemes? 
The docu is too fuzzy for me. Further when reading this I was not aware of any for loop whether double or not. 
Of course you know because you bisected the old code and sewed it into this "Frankenstein" ;-). 
However I don't know where the pieces come from.  Anyways if this is rather internal, I can live with never touching it again.